client/vip/controller.js

```diff
@@ -1,19 +1,19 @@
 /**
  * External Dependencies
  */
-var React = require( 'react' ),
-	i18n = require( 'i18n-calypso' ),
-	page = require( 'page' );
+import React from 'react';
+import i18n from 'i18n-calypso';
+import page from 'page';
 
 /**
  * Internal Dependencies
  */
-var setTitle = require( 'state/document-head/actions' ).setDocumentHeadTitle,
-	sites = require( 'lib/sites-list' )();
+import { setDocumentHeadTitle as setTitle } from 'state/document-head/actions';
+const sites = require( 'lib/sites-list' )();
```

Comment:
A pattern that's used in a few places for imports like `sites-list` looks something like this:
```js
import sitesFactory from 'lib/sites-list';
const sites = sitesFactory();
```
Though the naming isn't entirely consistent throughout the code base ([here](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/themes/themes-site-selector-modal.jsx), [here](https://github.com/Automattic/wp-calypso/blob/master/client/components/site-selector/add-site.jsx) & [here](https://github.com/Automattic/wp-calypso/blob/master/client/components/sites-dropdown/index.jsx) for examples)
Might it be worth making that change in this PR or leaving for another PR that targets this pattern specifically?